src/components/YearRecapCharts.tsx

```diff
@@ -101,7 +102,7 @@ export default function Example(props: BarsProps) {
                 strokeWidth={1}
                 orientation={['diagonal']}
             />
-            <rect x={0} y={0} width={props.width} height={props.height} fill='#eaeffb' rx={14} />
+            <rect x={0} y={0} width={props.width} height={props.height} key={props.id} fill='#eaeffb' rx={14} />
```

Comment:
React needs a unique key for each element rendered from a list. The list is graphDataAndLables. 
1. The rendered element here is everything within the fragment --> <><Bar><text></>
2. the other React caveat, the key should go on the outer most element in the list so you'll need to change the wrapping fragment add the key prop there. see doc for use https://react.dev/reference/react/Fragment#fragment
3. Can remove key from rect above - it is not being rendered in a list

also please remove unused variables/getters copied from visx doc, should change "Example" name.